dahee/models/userDao.js

```diff
@@ -18,6 +18,26 @@ const createUser = async (name, email, password, profileImage) => {
   }
 };
 
+const verifyUser = async (email) => {
```

Comment:
email로 user를 확인하니까 함수 이름을 다음과 같이 수정해도 좋을 것 같아요.
```suggestion
const verifyUserByEmail = async (email) => {
```